LayoutTests/platform/gtk/TestExpectations

```diff
@@ -1949,3 +1949,5 @@ http/tests/cookies/same-site/popup-cross-site-post.html [ Failure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/element-is-grouping-during-animation.html [ ImageOnlyFailure Pass ]
 
 webkit.org/b/213782 webanimations/accelerated-animation-with-easing.html [ Failure Pass ]
+
+webkit.org/b/286318 http/tests/iframe-monitor [ Skip ]
```

Comment:
To match the existing pattern, it's preferable to mark the test as `[ Skip ]` in the top-level TestExpectations file (which it looks like you're already doing), so you don't need to touch the gtk, win, and wpe TestExpectations files.

Assuming that this works on all wk2 ports, then the only platform-specific expectation change should be a `[ Pass ]` in the wk2 expectations file. If this only works in Cocoa WK2 ports, then you should add `[ Pass ]` to the `mac-wk2` and `ios` expectations files.

Reply:
Okay.